app/backend/api.ts

```diff
@@ -56,3 +57,13 @@ export const getPaymentDetails = async (payeeName: string, partPostcode: string)
 
 	return JSON.parse(response.payload)
 }
+
+
+export const getDownloadDetailsCsv = async (payeeName: string, partPostcode: string): Promise<Buffer> => {
+	const url = getUrlParams('downloaddetails', {
+		payeeName,
+		partPostcode
+	})
+	const content = await utils.getBuffer(url)
```

Comment:
I believe you can simplify this to
`return utils.getBuffer(url)` because the function is declared with an Async keyword

Reply:
true, updated the code.